etc/kayobe/ansible/deploy-baremetal-instance.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: Update network allocations for new hypervisors
+  hosts: compute
+  gather_facts: false
+  connection: local
+  serial: 1
+  vars:
+    network_allocation_path: "{{ kayobe_env_config_path }}/network-allocation.yml"
+  tasks:
+    - name: Show baremetal node
+      ansible.builtin.shell:
+        cmd: "openstack baremetal node show {{ inventory_hostname }} -f json"
+      register: node_show
+      failed_when: false
+      changed_when: false
+      delegate_to: localhost
+
+    - name: Set baremetal node JSON variable
+      ansible.builtin.set_fact:
+        node_show_json: "{{ node_show.stdout | to_json | from_json }}"
```

Comment:
`| to_json | from_json` is a bit weird, and the name of the variable doesn't really fit.

You're taking a string, which is JSON, then converting it to a nice ansible dict var, so "node_show_json" isn't actually json

Also, is it actually used? Can it just be deleted?